source/getting-started.txt

```diff
@@ -0,0 +1,72 @@
+.. _relational-migrator-getting-started:
+
+===============
+Getting Started
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+Relational Migrator allows you to bring your relational workloads 
+to MongoDB. 
+
+To get familiar with Relational Migrator capabilities, 
+features, and use cases review the follow resources:
+
+- :ref:`<supported-databases>`
+-  :ref:`<rm-migration-scenarios>`
+- `Relation Migrator Learning Byte <https://learn.mongodb.com/courses/mongodb-relational-migrator>`__
+- `Relational Migrator NYC.Local Product Demo <https://www.youtube.com/watch?v=-ND6_m397OM&ab_channel=MongoDB>`__
+
+Steps
+-----
+
+The following steps are a high level workflow of how you can use 
+Relational Migrator to get your data into MongoDB. These steps do not 
+cover all features or capabilities of the tool.
+
+.. procedure::
+
+   .. step:: Install
+
+      Relational Migrator supports 
+      :ref:`single machine installs <single-machine-install>` suitable 
+      for testing and :ref:`unattended server installs <unattended-server>` 
+      for production workloads.
+
+   .. step:: Create a project
+
+      Start using Relation Migrator by 
+      :ref:`creating a project <rm-projects>`. You can connect your 
+      source and destination databases with the appropriate 
+      :ref:`connection strings <rm-connection-strings>`.
```

Comment:
(more detail)
Part of a get started page is to introduce folks to the main concepts. You might add a sentence describing what a project is and then link out to how to create one. You can apply this throughout.

Reply:
I already have a link to the creating a project page, I did add a bit more detail around projects to this step and additional context to other steps